core/src/main/java/org/apache/iceberg/rest/responses/ConfigResponse.java

```diff
@@ -178,8 +199,14 @@ public Builder withEndpoints(List<Endpoint> endpointsToAdd) {
       return this;
     }
 
+    /** Sets the optional idempotency key lifetime advertised by the server. */
+    public Builder withIdempotencyKeyLifetime(@Nullable String lifetime) {
```

Comment:
I don't think we typically add `@Nullable` to method params

Reply:
Removed. Thanks